go/pkg/sysdb/coordinator/model/collection.go

```diff
@@ -21,6 +23,10 @@ type Collection struct {
 	TotalRecordsPostCompaction uint64
 	SizeBytesPostCompaction    uint64 // Note: This represents the size of the records off the log
 	LastCompactionTimeSecs     uint64
+	IsDeleted                  bool
+	VersionFileName            string
+	CreatedAt                  time.Time
```

Comment:
is created at different than the Ts field?

Reply:
Don't know. The person with that knowledge decided to leave us :(